src/domain/user.js

```diff
@@ -276,30 +331,3 @@ async function _findByUniqueAsATeacher(key, value) {
   }
   return null
 }
-
-async function _findByUnique(key, value) {
-  const foundUser = dbClient.user.findUnique({
-    where: {
-      [key]: value
-    },
-    select: {
-      id: true,
-      cohortId: true,
-      role: true,
-      email: true,
-      profile: {
-        select: {
-          firstName: true,
-          lastName: true,
-          bio: true,
-          githubUsername: true
-        }
-      }
-    }
-  })
-
-  if (foundUser) {
-    return foundUser
-  }
-  return null
-}
```

Comment:
this function should exist. it will return the student

also, yesterday or this morning we updated this function. i think is better to speak about this with our teachers.